presto-cli/src/test/java/com/facebook/presto/cli/TestQueryRunner.java

```diff
@@ -138,4 +124,25 @@ private String createResults()
                 null);
         return QUERY_RESULTS_CODEC.toJson(queryResults);
     }
+
+    static QueryRunner createQueryRunner(ClientSession clientSession)
```

Comment:
`Add test helper for creating QueryRunner`

IMO it is more QueryRunner factory method for tests.

Reply:
Good point, yes, "factory" is a better name than "helper"